python/examples/ingestion_with_yaml_config/telemetry_configs/nostromo_lv_426.yml

```diff
@@ -88,6 +88,30 @@ rules:
         - failure
         - nostromo
 
+  - namespace: voltage
```

Comment:
I ended up adding this `namespace` key because that made it obvious whether we should expect a fully formed rule or a reference to one in a separate namespace which just made validation a lot clearer, but definitely very open to feedback here